src/Builder/Visitor/AddMethodCallToCallListVisitor.php

```diff
@@ -136,6 +144,40 @@ protected function addNewCallIntoArrayMergeFuncNode(FuncCall $node, CallMetadata
         return $node;
     }
 
+    /**
+     * @param \PhpParser\Node\Expr\Array_ $node
+     * @param \SprykerSdk\Integrator\Transfer\CallMetadataTransfer $callMetadataTransfer
+     *
+     * @return \PhpParser\Node
+     */
+    protected function addNewCallIntoArray(Array_ $node, CallMetadataTransfer $callMetadataTransfer): Node
```

Comment:
is it copypasted? if yes you can move it to trait for now. because I saw this logic somewhere